app/modules/sightings/models.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Sightings database models
+--------------------
+"""
+
+from app.extensions import FeatherModel, db
+import uuid
+
+
+class Sighting(db.Model, FeatherModel):
+    """
+    Sightings database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+    title = db.Column(db.String(length=50), nullable=False)
+
+    def __repr__(self):
+        return (
+            '<{class_name}('
+            'guid={self.id}, '
```

Comment:
boilerplate gets this wrong, that should be self.guid

Reply:
Resolved.